src/components/Checkbox/Checkbox.svelte

```diff
@@ -15,20 +15,42 @@
   export let disabled = false;
   export let classes = classesDefault;
   export let labelClasses = i => i;
+  export let group = [];
 
+  // for bind:group
+  //keep track of group array state;
+  let groupstate = group.includes(value);
+
+  $: if (value && !disabled) {
```

Comment:
Wouldn't it be easier to use Set type instead all of this logic?

Reply:
Here is a repl which shows that forwarding group props from the input element does not seem to work properly, if that is what you mean: https://svelte.dev/repl/1565708677134e418e256234984d90ef?version=3.29.0